lib/iris/fileformats/__init__.py

```diff
@@ -28,7 +28,7 @@
 from . import abf
 from . import um
 try:
-    from . import grib as igrib
+    import iris_grib as igrib
```

Comment:
The import at this level means we have to be quite careful not to import this module when ```iris``` is imported, otherwise we would end up with circular imports for iris-grib. Long-term I'd be more in favour of a lazy import.